docker/docker_build.sh

```diff
@@ -82,545 +80,545 @@ CMAKE_VERSION=cmake-3.20.6-linux-x86_64
 CMAKE_VERSION_AARCH64=cmake-3.20.6-linux-aarch64
 
 print_usage_and_exit_docker_build() {
-    echo "$__usage_docker_build"
-    exit 1
+	echo "$__usage_docker_build"
```

Comment:
Shall we keep using (4) spaces as indentations, as we use in other places?

Reply:
Done